docs/4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_init.md

```diff
@@ -12,26 +12,39 @@ This command looks for a zarf-init package in the local directory that the comma
 
 
 
-Example Usage:
-# Initializing without any optional components:
+## Example Usage:
+
+### Initializing without any optional components:
+```shell
 zarf init
+```
 
-# Initializing w/ Zarfs internal git server:
+### Initializing w/ Zarfs internal git server:
+```shell
 zarf init --components=git-server
+```
 
-# Initializing w/ Zarfs internal git server and PLG stack:
+### Initializing w/ Zarfs internal git server and PLG stack:
+```shell
 zarf init --components=git-server,logging
+```
 
-# Initializing w/ an internal registry but with a different nodeport:
+### Initializing w/ an internal registry but with a different nodeport:
+```shell
 zarf init --nodeport=30333
+```
 
-# Initializing w/ an external registry:
+### Initializing w/ an external registry:
+```shell
 zarf init --registry-push-password={PASSWORD} --registry-push-username={USERNAME} --registry-url={URL}
+```
 
-# Initializing w/ an external git server:
+### Initializing w/ an external git server:
+```shell
 zarf init --git-push-password={PASSWORD} --git-push-username={USERNAME} --git-url={URL}
+```
 
-
+# General
```

Comment:
These docs are actually autogenerated with our `make docs-and-schema` make target: https://github.com/defenseunicorns/zarf/blob/main/Makefile#L98 from our CLI command code here: https://github.com/defenseunicorns/zarf/blob/main/src/cmd/initialize.go#L26 (and here is where the text comes from: https://github.com/defenseunicorns/zarf/blob/main/src/config/lang/english.go#L91)

We might be able to switch to the Cobra examples format like we do here: https://github.com/defenseunicorns/zarf/blob/main/src/cmd/package.go#L199 or just update the `english.go` file with these changes.

Reply:
ok sounds good, i'll close this out and look to update the go source.

Reply:
superceded by https://github.com/defenseunicorns/zarf/pull/1492